addon/components/cf-field/input/radio.js

```diff
@@ -0,0 +1,12 @@
+import Component from "@ember/component";
+import layout from "../../../templates/components/cf-field/input/radio";
+
+/**
+ * Input component for the radio question type
+ *
+ * @class CfFieldInputRadioComponent
```

Comment:
Can you tell me a bit about the comments? (`@class`, `@argument`, `@property` and so on)

Reply:
Those are standard YUIdoc tags along with some custom (just `@accessor` for now) tags used by [ember-cli-addon-docs-yuidoc](https://github.com/ember-learn/ember-cli-addon-docs-yuidoc)

Reply:
Can one see the documented documentation anywhere?